docs/topics/proc_add-debug-logging-spring-boot.adoc

```diff
@@ -0,0 +1,113 @@
+= Add Debug Logging
+There are several options for debug logging. This example uses the Arquillian Testrunner JUnit Container.
```

Comment:
Arquillian is totally unrelated, and this entire file doesn't really say anything about logging.

Reply:
@Ladicek This was just used to add debugging statements for the app. Can update the text.  Yes I will get a SME review as well thanks.